dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-sql.ts

```diff
@@ -43,6 +43,10 @@ export function useSql(model: { [field: string]: any }): IJsonItem[] {
         trigger: ['input', 'trigger'],
         required: true,
         message: t('project.node.sql_empty_tips')
+      },
+      props: {
+        language: 'sql',
+        placeholder: t('project.node.sql_statement_tips')
```

Comment:
It seems that the monaco editor does not support placeholder yet.

Reply:
@Amy0104 Thanks for the information. When I tested it manually, the placeholder didn't take effect. But highlight and auto-complete did work. What about removing the `placeholder` and only keeping the `language`?

Reply:
That's better to remove the placeholder.